src/test/java/be/seeseemelk/mockbukkit/entity/LivingEntityMockTest.java

```diff
@@ -0,0 +1,45 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+class LivingEntityMockTest
+{
+
+	private ServerMock server;
+	private CowMock livingEntity;
+
+	@BeforeEach
+	void setup()
+	{
+		server = MockBukkit.mock();
+		livingEntity = new CowMock(server, UUID.randomUUID());
+	}
+
+	@AfterEach
+	void tearDown()
+	{
+		MockBukkit.unmock();
+	}
+
+	@Test
+	void testIsJumpingDefault()
+	{
+		assertFalse(livingEntity.isJumping());
+	}
+
+	@Test
+	void testSetJumping()
+	{
+		livingEntity.setJumping(true);
```

Comment:
This test has no assertion.
I assume you meant to add `assertTrue(livingEntity.isJumping());` here.